MoviesMVVM/MoviesMVVM/MainModule/View/MoviesViewController.swift

```diff
@@ -3,6 +3,8 @@
 
 import UIKit
 
+typealias StringHandler = (String) -> Void
```

Comment:
```suggestion
typealias StringHandler = (String) -> Void
```
собери все в 1 м отдельном файле